js/server/server.js

```diff
@@ -41,6 +41,8 @@
     restServer = options["server.rest-server"];
   }
 
+  // make sure the cache is filled:
+  let col = internal.db._collections();
```

Comment:
This is not needed on the server-side, as there is no collection cache in arangod. Such cache exists in arangosh only to avoid frequent fetching of the list of collections.
```suggestion
```